### ISSUE TITLE
Admin UI : Top navbar titles that are too long will break UI

### ISSUE THREAD

**Issue by Skrypt** [OrchardCMS/OrchardCore#8084]
And make top navbar buttons unclickable.

![image](https://user-images.githubusercontent.com/3228637/102683676-a6909c80-41a0-11eb-94b4-85dee708a121.png)

**Comment by hishamco** [comment]
I think this is a common issue if the long title doesn't have a space

**Comment by Skrypt** [comment]
I'm just reporting issues that I find here. There is no issue with a long title when it's either ellipsed or overflow hidden or when using other techniques likes these. Also, depending where it's used these tricks are not necessary. I personally don't see why we added this title in the navbar. If it was because the title disapear when we scroll then it was still disapearing before and we did not had to have the title in the navbar. And also the browser tab name has the current section displayed in it's title. "Orchard - Manage Content" like here

![image](https://user-images.githubusercontent.com/3228637/102684260-fe310700-41a4-11eb-8490-ac18f7a6221b.png)
